platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -12,40 +12,41 @@ MGL_EXPORT
 @interface MGLMapSnapshotOptions : NSObject
 
 /**
- Creates a set of options with the minimum required information
+ Creates a set of options with the minimum required information.
+ 
  @param styleURL the style url to use
  @param camera the camera settings
  @param size the image size
  */
 - (instancetype)initWithStyleURL:(NSURL*)styleURL camera:(MGLMapCamera*)camera size:(CGSize)size;
 
-#pragma mark - Configuring the map
+#pragma mark - Configuring the Map
 
 /**
   The style URL for these options.
  */
-@property (nonatomic, readonly) NSURL* styleURL;
+@property (nonatomic, readonly) NSURL *styleURL;
 
 /**
- The zoom. Default is 0.
+ The zoom level. Default is 0.
  */
-@property (nonatomic) double zoom;
+@property (nonatomic) double zoomLevel;
 
 /**
  The `MGLMapcamera` options to use.
  */
-@property (nonatomic) MGLMapCamera* camera;
+@property (nonatomic) MGLMapCamera *camera;
 
 /**
- A region to capture. Overrides the center coordinate
- in the mapCamera options if set
+ The cooordinate rectangle that encompasses the bounds to capture. Overrides the center coordinate
+ in the mapCamera options if set.
```

Comment:
What constitutes an _unset_ MGLCoordinateBounds? A bounds with no area, or does it have to also be centered on Null Island?

Reply:
A bounds with no area, if coordinateBounds are not set it will use mapCamera. Would you like to complement this sentence to make this more clear?
> Overrides the center coordinate in the mapCamera options if set

Reply:
> The cooordinate rectangle that encompasses the bounds to capture.
> 
> If this property is non-empty and the `camera` property is non-nil, the camera’s center coordinate and altitude are ignored in favor of this property’s value.